src/content/recipes/@vanilla-extract/css.md

```diff
@@ -42,56 +42,19 @@ You can view a full example on [Stackblitz](https://stackblitz.com/edit/github-c
 
 ### Webpack
 
-For Webpack users, add a `webpackFinal` function to your `main.ts` file like so:
-
-```ts
-import type { StorybookConfig } from '@storybook/your-framework';
-
-import { VanillaExtractPlugin } from '@vanilla-extract/webpack-plugin';
-import MiniCssExtractPlugin from 'mini-css-extract-plugin';
-
-const config: StorybookConfig = {
-  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
-  addons: ['@storybook/addon-essentials'],
-  framework: {
-    name: '@storybook/your-framework',
-    options: {},
-  },
-  // 👇 Add this for Vanilla Extract
-  webpackFinal(config, options) {
-    // Add Vanilla-Extract and MiniCssExtract Plugins
-    config.plugins?.push(new VanillaExtractPlugin(), new MiniCssExtractPlugin());
-
-    // Exclude vanilla extract's "*.vanilla.css" files from other "*.css" processing
-    config.module?.rules?.forEach((rule) => {
-      if (typeof rule !== 'string' && rule.test instanceof RegExp && rule.test.test('test.css')) {
-        rule.exclude = /\.vanilla\.css$/i;
-      }
-    });
-
-    config.module?.rules?.push({
-      // Targets only CSS files generated by vanilla-extract
-      test: /\.vanilla\.css$/i,
-      use: [
-        MiniCssExtractPlugin.loader,
-        {
-          loader: require.resolve('css-loader'),
-          options: {
-            // Required as image imports should be handled via JS/TS import statements
-            url: false,
-          },
-        },
-      ],
-    });
-
-    return config;
-  },
-};
-
-export default config;
+For Webpack users, [@storybook/addon-styling-webpack](https://storybook.js.org/addons/@storybook/addon-styling-webpack/) can get your Storybook up and running with Vanilla Extract in a few lines of code.
```

Comment:
Is a section like this now appropriate in (some of) the other recipes?

Reply:
Definitely! My only concern is putting this in every relevant recipe before it's tested more thoroughly in the wild